angular-workspace/projects/ni/nimble-angular/src/directives/radio-button/nimble-radio-button.directive.ts

```diff
@@ -0,0 +1,24 @@
+import { Directive, ElementRef, Input, Renderer2 } from '@angular/core';
+import { RadioButton } from '@ni/nimble-components/dist/esm/radio-button';
+import type { RadioGroup } from '@ni/nimble-components/dist/esm/radio-group';
+import { BooleanValueOrAttribute, toBooleanProperty } from '../utilities/template-value-helpers';
+
+export { RadioButton };
```

Comment:
Should be able to `export type` instead of `export`. There isn't necessarily a compelling reason to do that other than consistency with the other directives. (if we start exporting the concrete type we should do that consistently)

Reply:
Exported as type